src/library/Cardano/CoinSelection/Fee.hs

```diff
@@ -98,20 +98,41 @@ newtype Fee = Fee { unFee :: Coin }
     deriving stock (Eq, Generic, Ord)
     deriving Show via (Quiet Fee)
 
--- | Defines the maximum size of a dust coin.
+-- | Defines the /maximum/ size of a __dust coin__.
 --
--- Change values that are less than or equal to this threshold will not be
--- included in coin selections produced by the 'adjustForFee' function.
+-- Functions that accept a 'DustThreshold' argument will generally exclude
+-- values that are /less than or equal to/ this threshold from the 'change'
+-- sets of generated selections, /coalescing/ such coins together into larger
+-- coins that /exceed/ the threshold.
+--
+-- Specifying a dust threshold of __/n/__ causes all coins that are less than
+-- or equal to __/n/__ to be treated as dust and coalesced together.
```

Comment:
you mean all coins are treated as dust and `subject to coalescence`, right?

Reply:
> you mean all coins are treated as dust and `subject to coalescence`, right?

So this specifically refers only to coins less than or equal to the threshold, rather than all coins.

If you're asking about the term "coalesced together", then I think this sounds more natural (and more direct) than coalescence (in this particular case).